pkg/blobfuse/blobfuse.go

```diff
@@ -160,9 +162,9 @@ func appendDefaultMountOptions(mountOptions []string) []string {
 
 // get storage account from secrets map
 // returns <accountName, accountKey, accountSasToken>
-func getStorageAccount(secrets map[string]string) (string, string, string, error) {
+func getStorageAccountFromSecretsMap(secrets map[string]string) (string, string, string, error) {
```

Comment:
add a unit test for getStorageAccountFromSecretsMap

Reply:
done.